Gems/EMotionFX/Code/EMotionFX/Pipeline/RCExt/Motion/MotionDataBuilder.cpp

```diff
@@ -237,9 +237,9 @@ namespace EMotionFX
             AZStd::vector<size_t> rootJoints; // The list of root nodes.
 
             // Data for root motion extraction
-            const size_t invalidJointDataIndex = AZStd::numeric_limits<size_t>::max();
-            size_t sampleJointDataIndex = invalidJointDataIndex;
-            size_t rootJointDataIndex = invalidJointDataIndex;
+            const size_t InvalidJointDataIndex = AZStd::numeric_limits<size_t>::max();
```

Comment:
Is this just locally used? We might turn this into a static / constexpr in case this is used anywhere else as well.